AVOutput/AVOutputTV.cpp

```diff
@@ -0,0 +1,5263 @@
+    }
+
+    static void tvVideoFrameRateChangeHandler(tvVideoFrameRate_t frameRate, void *userData)
+    {
+        LOGINFO("tvVideoFrameRateChangeHandler format:%d \n",frameRate);
+        AVOutputTV *obj = (AVOutputTV *)userData;
+        if(obj)obj->NotifyVideoFrameRateChange(frameRate);
+    }
+
+    static bool getVideoContentTypeToString(tvContentType_t content)
+    {
+        bool fmmMode = false;
+        switch(content) 
+	{
+            case tvContentType_FMM:
+                LOGINFO("Content Type: FMM\n");
+                fmmMode = true;
+                break;
+            default:
+                LOGINFO("Content Type: NONE\n");
+                fmmMode = false;
+                break;
+        } 
+        return fmmMode;
+    }
+
+    static const char *getVideoFormatTypeToString(tvVideoFormatType_t format)
+    {
+        const char *strValue = "NONE";
+        switch(format) 
+	{
+            case VIDEO_FORMAT_SDR:
+                LOGINFO("Video Format: SDR\n");
+                strValue = "SDR";
+                break;
+            case VIDEO_FORMAT_HDR10:
+                LOGINFO("Video Format: HDR10\n");
+                strValue = "HDR10";
+                break;
+            case VIDEO_FORMAT_HDR10PLUS:
+                LOGINFO("Video Format: HDR10PLUS\n");
+                strValue = "HDR10PLUS";
+                break;
+            case VIDEO_FORMAT_HLG:
+                LOGINFO("Video Format: HLG\n");
+                strValue = "HLG";
+                break;
+            case VIDEO_FORMAT_DV:
+                LOGINFO("Video Format: DV\n");
+                strValue = "DV";
+                break;
+            default:
+                LOGINFO("Video Format:: NONE\n");
+                strValue = "NONE";
+                break;
+        }
+        return strValue;
+    }
+
+    static std::string getVideoResolutionTypeToString(tvResolutionParam_t resolution)
+    {
+        std::string strValue = "NONE";
+        std::string interlaceValue = (resolution.isInterlaced) ? "i" : "p";
+        if ( resolution.resolutionValue != tvVideoResolution_NONE ) 
+	{
+            strValue = std::to_string(resolution.frameWidth) + "*" + std::to_string(resolution.frameHeight) + interlaceValue;
+        }
+        LOGINFO("Video Resolution:[%s]\n", strValue.c_str());
+        return strValue;
+    }
+
+    static const char *getVideoFrameRateTypeToString(tvVideoFrameRate_t frameRate)
+    {
+       const char *strValue = "NONE";
+       switch(frameRate) 
+       {
+            case tvVideoFrameRate_24:
+                LOGINFO("Video FrameRate: 24\n");
+                strValue = "24";
+                break;
+            case tvVideoFrameRate_25:
+                LOGINFO("Video FrameRate: 25\n");
+                strValue = "25";
+                break;
+            case tvVideoFrameRate_30:
+                LOGINFO("Video FrameRate: 30\n");
+                strValue = "30";
+                break;
```

Comment:
instrad of repeating in every case LOGINFO("Video FrameRate :%s \n",strValue ); at end

Reply:
done